src/main/java/de/tum/in/www1/artemis/repository/ExamLiveEventRepository.java

```diff
@@ -32,11 +32,26 @@ public interface ExamLiveEventRepository extends JpaRepository<ExamLiveEvent, Lo
             """)
     List<ExamLiveEvent> findAllByStudentExamIdOrGlobalByExamId(@Param("examId") Long examId, @Param("studentExamId") Long studentExamId);
 
+    /**
+     * Find all events for the given student exam in reverse creation order.
+     *
+     * @param studentExamId the id of the student exam
+     * @return a list of events
+     */
+    @Query("""
+            SELECT event
+            FROM ExamLiveEvent event
+            WHERE event.studentExamId = :studentExamId
+            ORDER BY event.id DESC
+            """)
+    List<ExamLiveEvent> findAllByStudentExamId(@Param("studentExamId") Long studentExamId);
+
     /**
      * Delete all events for the given exam.
      *
      * @param examId the id of the exam
      */
-    @Transactional // delete
+    @Transactional
+    // delete
```

Comment:
The comment explains why `@Transactional` is used here, we want to use this annotation as few as possible.
```suggestion
    @Transactional // delete
```

Reply:
Guess that was the auto code formatter. Really have to tweak it, sth. is not right